change/@fluentui-react-charting-cfa801e0-6573-413b-bb65-4c94f5e8ddbe.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Couple of bugs:
The text is not rendering properly in rtl mode.
Hover card disappears when mouse is hovered over it.